code/src/reducers/loader.js

```diff
@@ -0,0 +1,18 @@
+// The purpose of this reducer is to handle a global "Loading" state variable. When a fetch starts, the isLoading is set to true. When a fetch is completed, isLoading is set to false.
+
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+  isLoading: false
+}
+
+export const loader = createSlice({
+  name: 'loader',
+  initialState,
+  reducers: {
+    setLoading: (state, action) => 
+    {
```

Comment:
Being honest, I'm surprised your code works with these indentations - we have to be very aware about how arrow functions work.
If after `=>` there are is no code (in the same line of course), then it can be interpreted by JS that our function ended. Because of that, example below is far more safe :

```
setLoading: (state, action) => {
  state.isLoading = action.payload
}
```